src/app_osd.ts

```diff
@@ -25,7 +25,7 @@ export class AppOSD extends Viewer {
   private mapName: MapName | null = null;
   private listeners: ((isLoading: boolean) => void)[] = [];
 
-  constructor(mountTo: HTMLElement) {
+  constructor(mountTo: HTMLElement, useWebGL?: boolean) {
```

Comment:
Good work threading this through like you did. If this constructor gets bigger over time, it'll be a good call to turn it into an options bag like the App constructor.